microservices/services/contest-service/src/main/java/org/xcolab/service/contest/domain/contest/ContestDaoImpl.java

```diff
@@ -335,6 +347,74 @@ public boolean existsWithScheduleId(long contestScheduleId) {
 
     @Override
     public boolean delete(long contestPK) {
-        return dslContext.deleteFrom(CONTEST).where(CONTEST.CONTEST_PK.eq(contestPK)).execute() > 0;
+        final boolean[] result = new boolean[1];
```

Comment:
An `AtomicBoolean` would also be usable inside the lambda and might be nicer to read instead of the single-element array.

Reply:
Done.